crates/runtime/wit/deps/wasmcloud/host.wit

```diff
@@ -0,0 +1,14 @@
+package wasmcloud:host;
```

Comment:
this does not look right, I'd suggest `rm -rf wit/deps` and trying again from latest `main`